cli/index.ts

```diff
@@ -77,6 +78,11 @@ const OPTIONS: OptionConfigs = {
       "Which directory would you like the starter code folder to be in (default is current directory)?",
     choices: [],
   },
+  testing: {
+    id: "t",
+    description: "Test locally without cloning repo",
```

Comment:
Maybe an additional note that we expect a starter-code-v2 repo to already be in the outputDir?

Reply:
This is configurable in scrubberConfig.json so it can be run on any directory
I am planning to add some testing steps to the readme though